src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
@robinsch Not according to just about every other dummy effect; if nothing else it's a failsafe making sure we don't send off a nullptr to a spellcast that might rely on an actual reference should we for some odd reason not have one when we reach this point.